saltstack/states/github-actions/runner.sls

```diff
@@ -16,6 +18,7 @@ runner-mkdir:
 runner-download:
   archive.extracted:
     - name: /opt/actions-runner
+    # TODO: add conditional based on host architecture
```

Comment:
@rawkode I have no idea what I'm doing with salt... As far as I can tell, I can do something similar to the jinja template set below, using a if/else statement and get the host arch using grains['osarch'], but I also don't have a good way to test that 😂